packages/amazonq/package.json

```diff
@@ -125,6 +125,30 @@
                     "markdownDescription": "%AWS.configuration.description.amazonq.shareContentWithAWS%",
                     "default": true,
                     "scope": "application"
+                },
+                "amazonQ.localWorkspaceIndex": {
```

Comment:
why is `local` prefixing these settings? when would it be "non local" ? 

Local is the normal case. If there is a "remote" case, then it would make sense to name settings like `remoteFoo`. But in the normal case, it's redundant and confusing to mention "local".

Reply:
We are planning to do a remote indexing in the near future, this is why we name it local here.

Reply:
But again, "local" is the normal, common case. All settings are assumed to be "local". 

In the "remote" case, a "remote" qualifier makes sense. But that doesn't mean every setting needs a "local" qualifier.

Reply:
removed prefix "local"